cli/command/engine/activate.go

```diff
@@ -97,10 +98,23 @@ func runActivate(cli command.Cli, options activateOptions) error {
 			return err
 		}
 	}
-	if err = licenseutils.ApplyLicense(ctx, cli.Client(), license); err != nil {
+	dclient := cli.Client()
+	if err = licenseutils.ApplyLicense(ctx, dclient, license); err != nil {
 		return err
 	}
 
+	// Short circuit if the user didn't specify a version and we're already running enterprise
+	if options.version == "" {
+		serverVersion, err := dclient.ServerVersion(ctx)
+		if err != nil {
+			return err
+		}
+		if strings.Contains(strings.ToLower(serverVersion.Platform.Name), "enterprise") {
+			return nil
```

Comment:
We should either treat this as an error-situation, or inform the user that no changes were made. In the "happy path", a message will be printed "Successfully activated engine". If we're already on EE, I'd expect some form of feedback as well.

Reply:
sound good to make this change